stable/restore/README.md

```diff
@@ -150,11 +150,14 @@ The following tables list the configurable parameters of the restore chart and t
 
 | Parameter | Description | Default |
 | ----- | ----------- | ------ |
-| `restore.type` | What type of restore to perform: [ "database" | "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
-| `restore.target` | Where to restore `TO` | `{{ .Values.database.name }}` |
-| `restore.source` | Where to restore `FROM` [ backupset | url | `:latest` ] | `:latest` |
+| `restore.type` | What type of restore to perform: [ "database" \| "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
+| `restore.target` | The database name to request restore operations for | `"demo"` |
+| `restore.source` | The source which will be restored from. Supported values are [ _backupset_ \| _URL_ \| `:latest` \| `:group-latest` ]. URL restore source is expected to be in form of `protocol://authority/path`. Otherwise the value is interpreted as backupset name | `:latest` |
 | `restore.credentials` | Credentials to use for a URL source (user:password) | `""` |
 | `restore.autoRestart` | Whether to automatically restart the database and trigger the restore (true/false). Only valid for a "database" restore | `true` |
+| `restore.labels` | Process labels used to filter the complete set of archiveIds that should be restored which then defines the new state of the database upon restore. If multiple labels are defined, _any_ of them have to match so that the archive id is selected for restore. The setting works with NuoDB 4.2+ | `{ "backup": "cluster0" }` |
```

Comment:
Should we add a comment that a multi cluster MUST be modified away from the default in order to effect a full restore (e.g cluster1 must **also** be specified)?

Reply:
When restoring multi-cluster it's easier to just restore one of the clusters and let the other SMs sync from it. Alternatively, the "selected" backupset should be put on a remote location available to both clusters so that all SMs are restored.

I have some text about multi-cluster in the restore chart README. More should come in B&R documentation.

```
In multi-cluster deployments, different backup cron jobs execute with different schedules.
Specific backupsets are produced for database deployment in each cluster.
To perform a database restore in multi-cluster deployment, select the hotcopy SMs in one of the clusters and provide a backupset available in that cluster or `:group-latest` as `restore.source`.
```